### PR TITLE
[frenemies] Tweak UIs

### PR DESCRIPTION
**Pull request by Jordan-Mysten**
Some UI tweaks dealing with registration.
- Added spinner when loading
- Added new balance warning during registration
- Added new copy to connect wallet page.
- Removed dry run before execute.
- Tweak SUI balance formatting

Screenshot 2023-02-08 at 3 26 01 PM
Screenshot 2023-02-08 at 3 25 49 PM

### CODE REVIEW

File: dapps/frenemies/src/routes/Connect.tsx [hunk 1]
```diff
@@ -23,14 +47,38 @@ export function Connect() {
           Welcome to Sui Frenemies game
```
Comment (mamos-mysten) [inline dapps/frenemies/src/routes/Connect.tsx:47]: should this be `SUI` ?

File: dapps/frenemies/src/routes/Connect.tsx [hunk 2]
```diff
@@ -23,14 +47,38 @@ export function Connect() {
...
           Validator to one of three designated positions: Friend (top third),
-          Neutral (middle third), or Foe (bottom third).
+          Neutral (middle third), or Enemy (bottom third).
         </p>
+        <div className="h-px bg-steel/30 w-full my-8" />
+        <div className="text-heading6 text-steel-darker">
+          Here are a few things you will need to play this game.
+        </div>
+        <div className="mt-8 mb-12 grid grid-cols-1 sm:grid-cols-3 gap-x-10 gap-y-4">
+          <InfoItem title="A Sui Wallet">
```
Comment (mamos-mysten) [inline dapps/frenemies/src/routes/Connect.tsx:60]: same here

File: dapps/frenemies/src/routes/Connect.tsx (hunk 1 above)
Comment (Jordan-Mysten) [reply dapps/frenemies/src/routes/Connect.tsx:47]: I don't think so? I think we only use `SUI` for the token.

File: dapps/frenemies/src/routes/Connect.tsx (hunk 2 above)
Comment (Jordan-Mysten) [reply dapps/frenemies/src/routes/Connect.tsx:60]: Same answer

**Review by mamos-mysten — APPROVED** [review]
noticed some instances of `Sui` instead of `SUI` but not 100% sure what the rules are on that. other than that, lgtm

**Pull request merged**